test/serving.bash

```diff
@@ -105,7 +105,11 @@ function run_knative_serving_rolling_upgrade_tests {
   PROBER_PID=$!
 
   if [[ $UPGRADE_SERVERLESS == true ]]; then
-    serving_version=$(oc get knativeserving.operator.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.version}")
+    prev_serving_version=$(oc get knativeserving.operator.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.version}")
+    # This is ugly hack. Use KNATIVE_SERVING_VERSION if issues/361 was solved.
+    latest_serving_version=$(cat ${rootdir}/knative-operator/vendor/knative.dev/operator/version/version.go | grep "ServingVersion =" |  awk '{print $NF}' | tr -d '"')
```

Comment:
```suggestion
    latest_serving_version=$(sed -n 's/^.*ServingVersion.*"\(.*\)".*$/\1/p' knative-operator/vendor/knative.dev/operator/version/version.go)
```

I think this would be more robust (e.g. with respect to spacing)

Reply:
Thank you. Updated.